### ISSUE TITLE
Are there any plans to support direct mounting of aliyundrive using the ‘Aliyundrive Open Platform’ without the need for indirect mounting through webdav services like alist? 

### ISSUE THREAD

**Issue by 643104191** [rclone/rclone#7593]
# Are there any plans to support direct mounting of aliyundrive using the ‘Aliyundrive Open Platform’ without the need for indirect mounting through webdav services like alist? 

#### The associated forum post URL from `https://forum.rclone.org`

[post URL](https://forum.rclone.org/t/are-there-any-plans-to-support-direct-mounting-of-aliyundrive-using-the-aliyundrive-open-platform-without-the-need-for-indirect-mounting-through-webdav-services-like-alist/44189)

#### What is your current rclone version (output from `rclone version`)?

rclone v1.65.0
- os/version: Microsoft Windows 10 Pro 22H2 (64 bit)
- os/kernel: 10.0.19045.3930 (x86_64)
- os/type: windows
- os/arch: amd64
- go/version: go1.21.4
- go/linking: static
- go/tags: cmount

#### What problem are you are trying to solve?

The speed of accessing aliyundrive resources

#### How do you think rclone should be changed to solve that?

Add aliyundrive mounting method

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by kapitainsky** [comment]
You posted the same question on the forum already - which is right place for such discussion.

Please close it here.

**Comment by ncw** [comment]
I'm happy to keep this issue open as a wish list for new backends.

Though I think this will need an external contributor to provide the backend.

A bit of searching came up with https://github.com/K265/rclone/tree/aliyundrive-backend which looks like a backend in progress.

@K265 are you planning to submit this as a PR to rclone at some point?

**Comment by K265** [comment]
> I'm happy to keep this issue open as a wish list for new backends.
> 
> Though I think this will need an external contributor to provide the backend.
> 
> A bit of searching came up with https://github.com/K265/rclone/tree/aliyundrive-backend which looks like a backend in progress.
> 
> @K265 are you planning to submit this as a PR to rclone at some point?
> 
> 

Currently <https://github.com/K265/rclone/tree/aliyundrive-backend> only meet my daily needs and therefore lacks Open Api support. If someone is interested, welcome to PR!

@ncw I'd like to submit this as a PR when this feature is ready, thank you!

**Comment by ncw** [comment]
@K265 
> Currently https://github.com/K265/rclone/tree/aliyundrive-backend only meets my daily needs and therefore lacks Open Api support.

I'm not sure what that means? What does "lacks Open API support" imply? Are you using a reverse engineered API to access it>

> @ncw I'd like to submit this as a PR when this feature is ready, thank you!

Check out [these docs](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#writing-a-new-backend) as for the process. I'm happy to answer questions :-)

The most important thing is making the integration tests pass 100%. That will make it easy to merge.

**Comment by K265** [comment]
> @K265 
> > Currently https://github.com/K265/rclone/tree/aliyundrive-backend only meets my daily needs and therefore lacks Open Api support.
> 
> I'm not sure what that means? What does "lacks Open API support" imply? Are you using a reverse engineered API to access it>
> 
> > @ncw I'd like to submit this as a PR when this feature is ready, thank you!
> 
> Check out [these docs](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#writing-a-new-backend) as for the process. I'm happy to answer questions :-)
> 
> The most important thing is making the integration tests pass 100%. That will make it easy to merge.
> 
> 

Yes. Aliyundrive has introduced the new open APIs that allow third-party organizations to manage files, which are difference from the API extracted from their web pages.

When this feature is ready, I'll follow these steps, thanks :-)

**Comment by ypq123456789** [comment]
> > @K265
> > > Currently https://github.com/K265/rclone/tree/aliyundrive-backend only meets my daily needs and therefore lacks Open Api support.
> > 
> > 
> > I'm not sure what that means? What does "lacks Open API support" imply? Are you using a reverse engineered API to access it>
> > > @ncw I'd like to submit this as a PR when this feature is ready, thank you!
> > 
> > 
> > Check out [these docs](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#writing-a-new-backend) as for the process. I'm happy to answer questions :-)
> > The most important thing is making the integration tests pass 100%. That will make it easy to merge.
> 
> Yes. Aliyundrive has introduced the new open APIs that allow third-party organizations to manage files, which are difference from the API extracted from their web pages.
> 
> When this feature is ready, I'll follow these steps, thanks :-)

Thank you for all the hard work! I was wondering how the development of rclone's integration with Aliyun Disk Open API is progressing? I appreciate the effort to create a more direct connection.

**Comment by K265** [comment]
> > > @K265
> > > > Currently https://github.com/K265/rclone/tree/aliyundrive-backend only meets my daily needs and therefore lacks Open Api support.
> > > 
> > > 
> > > I'm not sure what that means? What does "lacks Open API support" imply? Are you using a reverse engineered API to access it>
> > > > @ncw I'd like to submit this as a PR when this feature is ready, thank you!
> > > 
> > > 
> > > Check out [these docs](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#writing-a-new-backend) as for the process. I'm happy to answer questions :-)
> > > The most important thing is making the integration tests pass 100%. That will make it easy to merge.
> > 
> > Yes. Aliyundrive has introduced the new open APIs that allow third-party organizations to manage files, which are difference from the API extracted from their web pages.
> > 
> > When this feature is ready, I'll follow these steps, thanks :-)
> 
> Thank you for all the hard work! I was wondering how the development of rclone's integration with Aliyun Disk Open API is progressing? I appreciate the effort to create a more direct connection.

Sorry, still have very little time on this.
